Libraries/LibLine/Editor.cpp

```diff
@@ -43,7 +43,6 @@ Editor::Editor()
     } else {
         m_num_columns = ws.ws_col;
         m_num_lines = ws.ws_row;
-        dbg() << m_num_lines;
```

Comment:
ouch, I removed this in my PR too...

Reply:
I'll rebase, no need to do it in a separate commit then :+1: